src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -57,62 +51,71 @@ public Manufacturer create(Manufacturer manufacturer) {
+                            .prepareStatement(sqlRequestGet)) {
             getByIdPreparedStatement.setLong(1, id);
             ResultSet resultSetGetById = getByIdPreparedStatement.executeQuery();
+            Manufacturer manufacturer = null;
             if (resultSetGetById.next()) {
-                String name = resultSetGetById.getString(NAME);
-                String country = resultSetGetById.getString(COUNTRY);
-                Long manufacturerId = resultSetGetById.getObject(ID, Long.class);
-                Manufacturer manufacturer = new Manufacturer(name, country);
-                manufacturer.setId(manufacturerId);
-                return Optional.of(manufacturer);
+                manufacturer = parserResultSet(resultSetGetById);
             }
-            return Optional.empty();
+            return Optional.ofNullable(manufacturer);
         } catch (SQLException throwables) {
             throw new DataProcessingException("Can`t get manufacturers from DB by id - "
-                    + id + " ", throwables);
+                    + id, throwables);
         }
     }
 
     @Override
     public Manufacturer update(Manufacturer manufacturer) {
-        sqlRequest = "UPDATE manufacturers SET name = (?), country = (?) "
+        String sqlRequestUpdate = "UPDATE manufacturers SET name = (?), country = (?) "
                 + "WHERE id = (?) AND is_deleted = false;";
         try (Connection connection = ConnectionUtil.getConnection();
                  PreparedStatement updatePreparedStatement = connection
-                            .prepareStatement(sqlRequest)) {
+                            .prepareStatement(sqlRequestUpdate)) {
             updatePreparedStatement.setString(1, manufacturer.getName());
             updatePreparedStatement.setString(2, manufacturer.getCountry());
             updatePreparedStatement.setLong(3, manufacturer.getId());
             updatePreparedStatement.executeUpdate();
             return manufacturer;
         } catch (SQLException throwables) {
             throw new DataProcessingException("Can`t update DB with manufacturer "
-                    + manufacturer + " ", throwables);
+                    + manufacturer, throwables);
         }
     }
 
     @Override
     public boolean delete(Long id) {
-        sqlRequest = "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
+        String sqlRequestDelete = "UPDATE manufacturers SET is_deleted = true WHERE id = ?;";
         try (Connection connection = ConnectionUtil.getConnection();
-                 PreparedStatement deleteByIdStatement = connection.prepareStatement(sqlRequest)) {
+                 PreparedStatement deleteByIdStatement = connection
+                         .prepareStatement(sqlRequestDelete)) {
             deleteByIdStatement.setLong(1, id);
-            return deleteByIdStatement.executeUpdate() >= 1;
+            return deleteByIdStatement.executeUpdate() >= 0;
```

Comment:
sorry, it was my mistake. We should use > instead of >=
```suggestion
            return deleteByIdStatement.executeUpdate() > 0;
```